src/main/java/com/github/markusbernhardt/seleniumlibrary/CustomRobotDriverElement.java

```diff
@@ -0,0 +1,48 @@
+package com.github.markusbernhardt.seleniumlibrary;
+
+import com.github.markusbernhardt.seleniumlibrary.keywords.BrowserManagement;
+import com.github.markusbernhardt.seleniumlibrary.keywords.Robot;
+import org.openqa.selenium.WebDriver;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.lang.reflect.Field;
+
+/**
+ * For extends of custom keywords of WebDriver actions
+ */
+public class CustomRobotDriverElement {
+
+    private static SeleniumLibrary s;
+    private static BrowserManagement b;
+    private Robot robot = new Robot();
+
+    public CustomRobotDriverElement() throws NoSuchFieldException, IllegalAccessException {
+        try {
+            CustomRobotDriverElement.s = getLibraryInstance();
+        } catch (javax.script.ScriptException e) {
```

Comment:
As we have an import for javax.script.ScriptException, we could use a short name here.

Reply:
Done